src/lib/modules/tests/solc_test.js

```diff
@@ -110,7 +110,7 @@ class SolcTest extends Test {
       },
       function run(contracts, web3, accounts, next) {
         let fns = [];
-        contracts.forEach((contract) => {
+        async.each(contracts, (contract, eachCb) => {
```

Comment:
Why switch to async.each if it's sync inside anyway? (you just push to the array)

Reply:
It was so the functions could be built up in parallel (though I doubt there is much performance benefit). I think I was desperately trying some async some stuff inside but must have removed it. Will revert as it's probably not doing much for us.